src/components/generic/client.hpp

```diff
@@ -5,7 +5,7 @@
 
 class GenericClient : public Generic {
    public:
-    ProtoType do_command(std::unordered_map<std::string, ProtoType*> command) override;
+    virtual AttributeMap do_command(AttributeMap command) override;
```

Comment:
(nit) I don't think we need to mark this as `virtual`, since it's already marked as an `override`